hardware/midi-keyboard/en.md

```diff
@@ -1,31 +1,22 @@
 +++
 title = "Using USB connected MIDI Keyboards"
-lastmod = "2019-04-14T11:20:00+01:00"
+lastmod = "2019-05-19T20:32:00+01:00"
 +++
 # Using USB connected MIDI Keyboards
 
 These instructions will guide you through how to set up and use a USB-connected MIDI Keyboard on Solus. This will easily let you use the keyboard to play music with a wide set of instruments.
 
 ## Installing required software
 
-First we'll need to install `qsynth` and `qjackctl` using the following command:
+We will need to install three packages in order to use the keyboard. These packages are, `qsynth` a frontend for fluidsynth, `qjackctl` a controller for the JACK audio server and a soundfont to use with fluidsynth.
 
 ``` bash
-sudo eopkg it qsynth qjackctl
+sudo eopkg it qsynth qjackctl fluid-soundfont
 ```
 
-Then we want to install a soundfont in order to get sounds and instruments that play when you press the keys on the keyboard.
+## Setting up the soundfont in qsynth
 
-``` bash
-mkdir -p $HOME/soundfonts
-wget http://http.debian.net/debian/pool/main/f/fluid-soundfont/fluid-soundfont_3.1.orig.tar.gz -O $HOME/soundfonts/fluid-soundfont.tar.gz
-cd $HOME/soundfonts && tar xvf fluid-soundfont.tar.gz fluid-soundfont-3.1/FluidR3_GM.sf2 --strip=1
-rm fluid-soundfont.tar.gz
-```
-
-## Setting up the soundfonts in qsynth
-
-In order to use the soundfonts we need to open up `qsynth` and set the soundfont `FluidR3_GM.sf2` (or any other soundfont) to be used. You do this by pressing `Setup...` and navigating to `Soundfonts`. In there you just click open and select your soundfont (should be placed in `~/soundfonts/`).
+In order to use the soundfont we need to open up `qsynth` and set the soundfont to be `FluidR3.sf2` in our case, but any other should work fine too. You do this by pressing `Setup...` and navigating to `Soundfonts`. There you just click `Open...` and navigate to the soundfont. The soundfont that we will be using is placed in `/usr/share/sounds/sf2/`.
```

Comment:
Would probably read better if it was something along the lines of:

```
In order to use the soundfont, we need to open up `qsynth` and set the soundfont. For the purposes of this article, it would be `FluidR3.sf2`, however any other should work fine too.
```